extensions/wikia/ArticleVideo/scripts/featured-video.jwplayer.on-scroll.js

```diff
@@ -0,0 +1,111 @@
+define('wikia.articleVideo.featuredVideo.jwplayer.onScroll', ['wikia.onScroll'], function (onScroll) {
+	return function (playerInstance, $featuredVideo, $playerContainer) {
+		var videoCollapsed = false,
+			collapsingDisabled = false,
+			$closeBtn = $('.featured-video__close');
+
+		// TODO move to separated module
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AS1135)